code/modules/integrated_electronics/core/printer.dm

```diff
@@ -1,35 +1,61 @@
-
 /obj/item/integrated_circuit_printer
 	name = "integrated circuit printer"
 	desc = "A portable(ish) machine made to print tiny modular circuitry out of metal."
 	icon = 'icons/obj/integrated_electronics/electronic_tools.dmi'
 	icon_state = "circuit_printer"
 	w_class = ITEMSIZE_LARGE
-	var/metal = 0
-	var/max_metal = 100
-	/// One sheet equals this much metal.
-	var/metal_per_sheet = 10
-	/// If true, metal is infinite.
-	var/debug = FALSE
-	/// When hit with an upgrade disk, will turn true, allowing it to print the higher tier circuits.
-	var/upgraded = FALSE
-	/// Same for above, but will allow the printer to duplicate a specific assembly. (Not implemented)
-	var/can_clone = FALSE
-//	var/static/list/recipe_list = list()
-	/// Not implemented :(
-	var/obj/item/electronic_assembly/assembly_to_clone = null
+	var/cur_metal = 0
+	var/max_metal = 250
+	var/metal_per_sheet = 10	// One sheet equals this much metal. was 10
```

Comment:
Why are we reverting autodoc formatting? The `///` a line before allows you to peek a description of a var Ctrl+Mouse hover

Reply:
And that applies to vars?! Nice! Okay, I'll go through and update all of the vars and such then.